presto-verifier/README.md

```diff
@@ -0,0 +1,51 @@
+Presto Verifier
+===============
+
+The Presto Verifier can be used to test Presto against a database, such
+as MySQL, or to test two Presto clusters against each other. We use it
+to continuously test the master branch against the previous release,
+while developing Presto.
+
+To use the Verifier, create a MySQL database with the following table
+and load it with the queries you would like to run:
+
+``` {.sql}
```

Comment:
remove {.sql}

Or replace with correct highlighting - https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting

Reply:
Verified that sql is supported: https://support.codebasehq.com/articles/tips-tricks/syntax-highlighting-in-markdown

Reply:
Yeah .. but the syntax is different...